### PR TITLE
Adding functionality to job

### PR DESCRIPTION
**Pull request by Navneet78**
I have created all methods for job.

### CODE REVIEW

File: pybuildkite/client.py [hunk 1]
```diff
@@ -100,6 +100,20 @@ def put(self, url, body=None, headers=None, query_params=None):
 
         return self.request("PUT", url=url, query_params=query_params, body=body, headers=headers)
 
+    def delete(self, url, body=None, headers=None, query_params=None):
```
Comment (pyasi) [inline pybuildkite/client.py]: DELETE is also being created https://github.com/pyasi/pybuildkite/pull/21 so we'll need to be smart about how we're merging.

CC: @savithakj

File: pybuildkite/jobs.py [hunk 2]
```diff
@@ -61,3 +61,48 @@ def get_job_environment_variables(self, organization, pipeline, build, job):
...
+           :fields: name and email
+           :unblocker: The user id of the person activating the job
+           :return: response
+        """
+        body ={
+            "field" : fields,
+            "unblocker" : unblocker
+        }
+        return self.client.put(
+            self.path.format(organization, pipeline, build, job)+"unblock", body=body
```
Comment (pyasi) [inline pybuildkite/jobs.py]: Two things. won't it need to be `/unblock`? And can we follow the same structure here https://github.com/pyasi/pybuildkite/blob/master/pybuildkite/builds.py#L259 just for consistency (minor change).

Thanks!

**Comment by Navneet78** [comment]
what do you mean by  _run black_ ?

**Comment by pyasi** [comment]
It’s a code linter/styler. Its in the requirements.txt

**Comment by Navneet78** [comment]
yeah I run black on job.py it remains unchanged so can you please merge and close this.

**Pull request merged**